docs/upgrade.html

```diff
@@ -41,9 +43,73 @@ <h5><a id="upgrade_420_notable" href="#upgrade_420_notable">Notable changes in 4
         The <code>PARTITIONER_ADPATIVE_PARTITIONING_ENABLE_CONFIG</code> in <code>ProducerConfig</code> was deprecated and will be removed in Kafka 5.0. Please use the <code>PARTITIONER_ADAPTIVE_PARTITIONING_ENABLE_CONFIG</code> instead.
     </li>
     <li>
-        The <code>ConsumerPerformance</code> command line tool has a new <code>include</code> option that is alternative to the <code>topic</code> option.
+        The <code>ConsumerPerformance</code> command line tool has a new <code>--include</code> option that is an alternative to the <code>--topic</code> option.
         This new option allows to pass a regular expression specifying a list of topics to include for consumption, which is useful to test consumer performance across multiple topics or dynamically matching topic sets.
     </li>
+    <li>
+        The consistency of argument names for the command-line tools has been improved (<a href="https://cwiki.apache.org/confluence/x/DguWF">KIP-1147</a>).
+        The deprecated options will be removed in Kafka 5.0.
+        <ul>
+            <li>
+                In <code>kafka-producer-perf-test.sh</code>, <code>--bootstrap-server</code> and <code>--reporting-interval</code> option are added.
+            </li>
+            <li>
+                In <code>kafka-console-consumer.sh</code> and <code>kafka-console-share-consumer.sh</code>, the option <code>--property</code>
+                which is used to specify the properties for the formatter is deprecated in favor of <code>--formatter-property</code>.
+            </li>
+            <li>
+                In <code>kafka-consumer-perf-test.sh</code> and <code>kafka-share-consumer-perf-test.sh</code>, the option <code>--messages</code>
+                is deprecated in favor of <code>--num-records</code> to bring all performance testing tools in line.
+            </li>
+            <li>
+                The option <code>--command-property</code> is used for all command-line tools which accept configuration properties directly on
+                the command line. The tools affected are:
+                <ul>
+                    <li>
+                        <code>kafka-console-consumer.sh</code>, <code>kafka-console-producer.sh</code> and
+                        <code>kafka-console-share-consumer.sh</code> (<code>--consumer-property</code> and <code>--producer-property</code> are
+                        deprecated in favor of <code>--command-property</code>)
+                    </li>
+                    <li>
+                        <code>kafka-producer-perf-test.sh</code> (<code>--producer-props</code> is deprecated in favor of <code>--command-property</code>)
+                    </li>
+                    <li>
+                        <code>kafka-consumer-perf-test.sh</code> and <code>kafka-share-consumer-perf-test.sh</code> gain
```

Comment:
```suggestion
                        <code>kafka-consumer-perf-test.sh</code> and <code>kafka-share-consumer-perf-test.sh</code> again
```
<img width="729" height="79" alt="Screenshot 2025-08-28 at 16 11 43" src="https://github.com/user-attachments/assets/d3ee5fb3-006c-4e79-95c7-421c2ff05dfe" />